service/mysql-deployment.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: mysql 
+  template:
+    metadata:
+      labels:
+        app: mysql
+    spec:
+      containers:
+      - args:
+        - --default-authentication-plugin=mysql_native_password
+        env:
+        - name: MYSQL_DATABASE
+          value: femiwiki
+        - name: MYSQL_PASSWORD
```

```suggestion
        # NOTE: 개발용으로만 사용되는 옵션이고 실제 프로덕션에서는 무시되는 옵션들임
        - name: MYSQL_PASSWORD
```

보고 놀랄수도 있는 사람들을 위해 주석 추가